extension/content.js

```diff
@@ -12,8 +12,12 @@ const getUsername = () => $('meta[name="user-login"]').attr('content');
 function linkifyBranchRefs() {
 	$('.commit-ref').each((i, el) => {
 		const parts = $(el).find('.css-truncate-target');
-		let username = parts.eq(0).text();
-		let branch = parts.eq(1).text();
+		let branch = parts.eq(parts.length - 1).text();
+
+		// if there are two parts the first part is the username
+		if (parts.length > 1) {
+			let username = parts.eq(0).text();
```

Comment:
Pulling this out will conflict with the global `const` for `username`. Maybe change the global to "ownername" and use "username" inside this function? Something like:

```js
const ownername = path.split('/')[1];
...
let username = ownername;
if (parts.length > 1) {
    username = parts.eq(0).text();
}
```

Then I'll clean up the other `if (!branch)` block which is also not necessary anymore.